src/server/server_family.cc

```diff
@@ -51,7 +52,9 @@ using namespace std;
 
 ABSL_FLAG(string, dir, "", "working directory");
 ABSL_FLAG(string, dbfilename, "dump", "the filename to save/load the DB");
-ABSL_FLAG(string, requirepass, "", "password for AUTH authentication");
+ABSL_FLAG(string, requirepass, "",
+          "password for AUTH authentication. if DFLY_PASSWORD env var is set, "
```

Comment:
The preference rule contradicts my intuition. 
I would expect that the flag will override environment behavior.